### PR TITLE
Update bouncy castle versions from v1.75 to 1.78.1

### LINKED ISSUE #28307: Update bouncycastle binaries from 1.75 to 1.78.1
Update open liberty projects to use the latest bcpkix, bcprov and bcutil jars version (version 1.78)

bcpkix: https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk18on/1.78
pcprov: https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk18on/1.78
bcutil: https://mvnrepository.com/artifact/org.bouncycastle/bcutil-jdk18on/1.78

### PR DESCRIPTION
**Pull request by saumyapandyaa**
Fixes https://github.com/OpenLiberty/open-liberty/issues/28307

### CODE REVIEW

**Comment by saumyapandyaa** [comment]
#libby
#build

**Comment by saumyapandyaa** [comment]
#libby
#build

**Comment by saumyapandyaa** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.microprofile.rest.client_fat,someOtherFAT,etc

**Comment by saumyapandyaa** [comment]
#libby
#build

**Comment by saumyapandyaa** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.microprofile.rest.client_fat,someOtherFAT,etc

**Comment by saumyapandyaa** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.microprofile.rest.client_fat,someOtherFAT,etc

**Comment by jjiwooLim** [comment]
@saumyapandyaa This build is failing because of `com.ibm.ws.install` has compilation failure. I had to add bcutil dependency and created [PR](https://github.com/OpenLiberty/open-liberty/pull/28376). If you want, you can take out the changes made to install package, or wait until I merge mine

**Comment by saumyapandyaa** [comment]
@jjiwooLim Thank you, that was helpful!

**Comment by saumyapandyaa** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.microprofile.rest.client_fat,someOtherFAT,etc

**Comment by jjiwooLim** [comment]
@saumyapandyaa my fix is in now, so you can take out your changes made in dev/com.ibm.ws.install/bnd.bnd

**Comment by saumyapandyaa** [comment]
#libby
#build
#spawn.fullfat.buckets=[com.ibm.ws](http://com.ibm.ws/).jaxrs.2.1.client_fat,[com.ibm.ws](http://com.ibm.ws/).jaxrs.2.0.client_fat,[com.ibm.ws](http://com.ibm.ws/).microprofile.rest.client_fat,[com.ibm.ws](http://com.ibm.ws/).install.featureUtility_fat

**Comment by saumyapandyaa** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.jaxrs.2.1.client_fat,com.ibm.ws.jaxrs.2.0.client_fat,com.ibm.ws.microprofile.rest.client_fat,com.ibm.ws.install.featureUtility_fat

**Comment by saumyapandyaa** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.jaxrs.2.1.client_fat,com.ibm.ws.jaxrs.2.0.client_fat,com.ibm.ws.microprofile.rest.client_fat,com.ibm.ws.install.featureUtility_fat

**Review by Zech-Hein — DISMISSED** [review]
LGTM

**Comment by saumyapandyaa** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.jaxrs.2.1.client_fat,com.ibm.ws.jaxrs.2.0.client_fat,com.ibm.ws.microprofile.rest.client_fat,com.ibm.ws.install.featureUtility_fat

File: dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml
Comment (jhanders34) [inline dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml:2542]: You should run `./gradlew cnf:dependabotGenerate` to update this file and put the file in.  In reality these entries go away since they are already there after these entries.

File: dev/com.ibm.ws.install/bnd.bnd
Comment (jhanders34) [inline dev/com.ibm.ws.install/bnd.bnd]:
```suggestion
```
This change is not needed since it was already done with the install PR.

**Review by jhanders34 — CHANGES_REQUESTED** [review]

File: dev/com.ibm.ws.install/bnd.bnd
Comment (jhanders34) [reply dev/com.ibm.ws.install/bnd.bnd]: You didn't remove these duplicate buildpath entries as I was saying in the review comment.  Not sure why you marked it resolved.

**Review by jhanders34 — APPROVED** [review]
I applied the change I requested into a new commit in the PR and as such I can approve the PR now.  You will need to pull your branch to bring it in locally if you want to have it in your local fork on your laptop.

**Comment by jhanders34** [comment]
#libby #build

**Comment by Zech-Hein** [comment]
PB results LGTM.

**Comment by clarkek123** [comment]
Hi @Zech-Hein and @saumyapandyaa Did all of the commits for this make the GM driver for 24.0.0.6?

**Comment by jhanders34** [comment]
> Hi @Zech-Hein and @saumyapandyaa Did all of the commits for this make the GM driver for 24.0.0.6?

[05bac7e](https://github.com/OpenLiberty/open-liberty/commit/05bac7e62ba5709cbf1115c7864a77e5c8704a28) has the [gm-24.0.0.6](https://github.com/OpenLiberty/open-liberty/releases/tag/gm-24.0.0.6) tag

**Comment by Zech-Hein** [comment]
> Hi @Zech-Hein and @saumyapandyaa Did all of the commits for this make the GM driver for 24.0.0.6?

yes ✅

**Pull request merged**
